lib/index.js

```diff
@@ -71,6 +71,19 @@ Strategy.prototype.authenticate = function (req, options) {
   if (req.query && req.query.error) {
     return this.fail(req.query.error);
   }
+
+  if (req.query.code) {
+    // The code param is sent to the /callback route
+    // where code exchange and other verification will happen
+    this._verify = verifyWrapper(this._verify, this.options, req.session.authParams);
```

Comment:
I also get a "verifyWrapper is not defined" so `var verifyWrapper = require('./verifyWrapper');` as well